app/services/recordings_sync.rb

```diff
@@ -6,7 +6,7 @@ def initialize(params)
   end
 
   def call
-    rooms = @user.rooms
+    rooms = Room.all # TODO: Revisit this.
```

Comment:
Uhhh I dont think this is a good idea

Reply:
Me either, it's just temporary.
Implementing it correctly requires more work since it may introduce its own infrastructure (ActiveJobs infrastructure, bulked batch updates,...).

Reply:
Let's leave it as is then

Reply:
The server recordings will only display the user recordings that clicked the re-sync button and not all of the rooms recordings.
Is that okay for now?

Reply:
That's fine